src/components/Footer.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+export const Footer = () => {
+
+  return (
+    <footer>Made by Us</footer>
+  );
+};
```

Comment:
this could probably be a one-line function, since it's simply returning this small element